pkg/bucketeer/user/user.go

```diff
@@ -9,8 +9,15 @@ type User struct {
 	Data map[string]string `json:"data,omitempty"`
 }
 
+func NewUser(id string, attributes map[string]string) *User {
```

Comment:
Although I plan to define NewUser in bucketeer package, it's not possible because of cycle dependency.